spec/inject_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'inject'
+
+describe Array do
+  it 'sums numbers' do
+    expect([4,2,3,9].new_inject(0) {|res,ele| res + ele}).to eq 18
+    expect([1,2,3,4].new_inject(0) {|res,ele| res + ele}).to eq 10
```

Comment:
Space missing after comma.<br>Space between { and | missing.<br>Space missing inside }.